rpm/matcher.go

```diff
@@ -0,0 +1,34 @@
+package rpm
+
+import (
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/libvuln/driver"
+
+	version "github.com/knqyf263/go-rpm-version"
+)
+
+// Matcher implements driver.Matcher.
+type Matcher struct{}
+
+var _ driver.Matcher = (*Matcher)(nil)
+
+func (*Matcher) Filter(record *claircore.IndexRecord) bool {
+	// record.Repository.Kind == "rpm"
+	return true
+}
+
+func (*Matcher) Query() []driver.MatchExp {
+	return []driver.MatchExp{
+		driver.PackageDistributionCPE,
+	}
+}
+
+func (*Matcher) Vulnerable(record *claircore.IndexRecord, vuln *claircore.Vulnerability) bool {
```

Comment:
I'd put a little comment here just explaining when/why we'd want out the cmp function.

Reply:
OK